QuizApp/Login/LoginViewController.swift

```diff
@@ -30,6 +32,7 @@ class LoginViewController: UIViewController {
     }
 
     @objc func handleLogIn() {
+        loginViewModel.login(username: emailTextField.text!, password: passwordTextField.text!)
```

Comment:
I know you're sure there's going to be something here, but it would be better to unwrap it with guard.